src/atomate2/vasp/sets/defect.py

```diff
@@ -44,13 +43,16 @@ def get_incar_updates(
             A dictionary of updates to apply.
         """
         return {
-            "IBRION": 2,
+            "IBRION": 1,
             "ISIF": 2,
             "EDIFFG": -0.05,
             "LREAL": False,
             "NSW": 99,
-            "LCHARG": False,
             "ENCUT": 500,
             "LAECHG": False,
             "NELMIN": 6,
+            "ALGO": "Normal",
+            "NCORE": 4,
+            "GGA": None,
```

Comment:
I don't think this should be here either? We want to set GGA/use the GGA tag from the previous calculation